### ISSUE TITLE
How to make Linux AppImage version run in portable mode?

### ISSUE THREAD

**Issue by Curculigo** [SpartanJ/ecode#351]
https://github.com/SpartanJ/eepp/releases/download/nightly/ecode-linux-nightly-x86_64.AppImage

Does portable_mode.txt work? If it does, where is the config files stored? The current directory where the AppImage is located?

**Comment by SpartanJ** [comment]
Yeah, not possible by design, there's no place to store the data safely, *but* portable mode still works with the app image, the data will be written in the default temp directory (for example that's usually `/tmp/ecode_portable`).
